.gitignore

```diff
@@ -101,11 +101,37 @@ ENV/
 .mypy_cache/
 
 # input data, saved log, checkpoints
-data/
 input/
 saved/
 datasets/
 
+# testing
+testing/
+test.py
+test_load.py
+test_notebook.ipynb
+test_utils.py
+best_run_numpy_analysis.py
+best_run_reloading.py
+best_run_reloading_analysis.py
+replication_run_reloading.py
+replication_run_analysis_old_framework.py
+replication_run_analysis_new_framework.py
+replication_plotting_new_framework.ipynb
+replication_plotting_old_framework.ipynb
+replication_plotting_test.ipynb
+multi_plotting_utils.py
+multi_run_plotting.py
+test_reloading.py
+test_example_config.yaml
+replication_plotting.ipynb
+repeating_classifier_training.py
+overlaid_roc.png
+ddp_run_analysis.py
+ddp_run_output_comparison.py
+test_reloading.py
+ip.txt
```

Comment:
Generally you want to keep this to single directories or file extensions, to avoid maintaining these longs lists of individual files. Can these be moved into a directory?

Reply:
Yeah that can be done, it was kind of ad hoc